model/vsc_parser.py

```diff
@@ -371,28 +412,44 @@ def ast_Properties(parent, yamltree) -> list[Property]:
     for st in subtrees:
         node = Property(get_yaml_value(st, 'name'), parent)
         node.description = get_yaml_value(st, 'description')
-        node.type = get_yaml_value(st, 'type')
```

Comment:
I notice that "type" (i.e. sensor/attribute (and actuator?)) has been removed from the vsc README for properties, but is still used within the VSC seat example. If that is intentional we shall better remove it from the example.

(Or do we need to have something similar to attribute/sensor/actuator for properties in VSC, if not using those words maybe something like r/rw. We say in readme that "Each properties list object specifies a shared state object that can be read and set, and which is available to all subscribing entities.", so it seems that properties shall be writeable)